community/duo_unix/APKBUILD

```diff
@@ -1,39 +1,27 @@
-# Contributor: Alan Lacerda <alacerda@alpinelinux.org>
-# Maintainer: Paul Morgan <jumanjiman@gmail.com>
+# Contributor: Kris Reese <ktreese@gmail.com>
+# Maintainer: Kris Reese <ktreese@gmail.com>
 pkgname=duo_unix
-pkgver=1.11.0
-pkgrel=1
-pkgdesc="duosecurity.com two-factor authentication"
-url="https://duo.com/support/documentation/duounix"
+pkgver=1.11.1
+pkgrel=0
+pkgdesc="Duo two-factor authentication for Unix systems with Pluggable Authentication Modules (PAM)"
+url="https://duo.com/docs/duounix"
 arch="all"
-license="GPL-2.0-only"
-depends=""
-depends_dev="openssl-dev zlib-dev"
-makedepends="$depends_dev automake autoconf libtool"
-options="suid"
-subpackages="$pkgname-dev $pkgname-doc"
-source="https://github.com/duosecurity/duo_unix/archive/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgname-$pkgver
+license="GPL-2.0"
+depends="openssh openssh-server-pam"
+makedepends="openssl-dev linux-pam-dev openssh-server-pam libtool"
+install=""
+subpackages="$pkgname-doc"
+source="https://dl.duosecurity.com/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+options="suid !check"
```

Comment:
Why no check? It needs comment or add check, basically to make sure that package works

Reply:
Resolved via fb84c2c325

Reply:
Readded `!check` as check consistently fails with:

```
In file included from /usr/include/fortify/stdio.h:20:0,
                 from testpam_preload.c:16:
testpam_preload.c:107:1: error: redefinition of 'fopen'
 fopen64(const char *filename, const char *mode)
 ^
/usr/include/stdio.h:63:7: note: previous definition of 'fopen' was here
 FILE *fopen(const char *__restrict, const char *__restrict);
       ^~~~~
make[2]: *** [Makefile:463: testpam_preload.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/builder/duo_unix/src/duo_unix-1.11.1/tests'
make[1]: *** [Makefile:652: check-am] Error 2
make[1]: Leaving directory '/home/builder/duo_unix/src/duo_unix-1.11.1/tests'
make: *** [Makefile:469: check-recursive] Error 1
make: Leaving directory '/home/builder/duo_unix/src/duo_unix-1.11.1'
>>> ERROR: duo_unix*: check failed
>>> ERROR: duo_unix: all failed
```

Any suggestions on how to resolve this?